src/docker/start.sh

```diff
@@ -1,20 +1,22 @@
-#!/bin/sh
-
-if ! [ -z "$CONF_FILE_PATH" ] && ! [ -f "CONF_FILE_PATH" ]; then
+#!/usr/bin/env bash
+if [[ -n "$CONF_FILE_PATH" ]] && [ ! -f "CONF_FILE_PATH" ]; then
```

Comment:
Doesn't work. If the file already exists, it stills goes to `Generating config file`